constantinCurtis/src/main/java/generics/first/task/Print.java

```diff
@@ -0,0 +1,17 @@
+package generics.first.task;
+
+public class Print {
+    public <T, S> void printArray(T[] t, S[] s) {
```

Comment:
please overload this method with single parameter, use `Arrays.toString()`

Reply:
Thanks for help, fixed